### DISCUSSION TITLE
[Feature Request] Button Alert for New Game Content

### DISCUSSION THREAD

**Discussion by StingX2** [RetroAchievements/RAWeb#811] [Ideas]
The idea is that when new content is added to a game developer (and up) could push a button that would alert users thru the inbox that played that Game ID of new content being added. Maybe even a way to alert how many achievements by inputting a number. One of the more common complaints is gamers are unaware when a lot of these revisions happen unless they stalk the discord or forum.

![chrome_XtmAYpVWHR](https://user-images.githubusercontent.com/70493569/126651536-d9ee73e9-003c-4a7b-99c2-5a694a316a29.png)

That is how TrueTrophies handles it. The difference is actual game publishers are releasing stuff and well thats it. The way our site works automated alerts any time a new achievement went live would be a disaster due to ticket takedowns or demotions. It'd be better to let the dev flip the on switch to send that email out because they would know when the game is ready to be RE-mastered.

I see this being a real benefit as we start to progress to features like Badge/Game Awards being reliant on keeping said achievements instead of a permanent status.

**Comment by StingX2** [comment]
As an addendum, I'd suggest there be an option in the user settings to select a threshold for these notifications. As an example, players can choose to only be notified of revisions if they have 50% or even 100% of the achievements unlocked in the set. This way, people can be notified of updates to sets that they enjoyed playing, but not games they played once and have no intention to finish.

**Comment by jeffjenx** [comment]
It might be better to implement a "watch" feature where users can select which games they want to watch for updates and be notified whenever something changes for that selection. I'm thinking something similar to GitHub's watch and star (aka favorite) features.
